extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -364,17 +365,16 @@ export class SafeframeHostApi {
    * @private
    */
   formatGeom_(iframeBox) {
-    const ampAdBox = this.baseInstance_.getPageLayoutBox();
     const viewportSize = this.viewport_.getSize();
     const currentGeometry = /** @type {JsonObject} */({
       'windowCoords_t': 0,
       'windowCoords_r': viewportSize.width,
       'windowCoords_b': viewportSize.height,
       'windowCoords_l': 0,
-      'frameCoords_t': ampAdBox.top,
-      'frameCoords_r': ampAdBox.right,
-      'frameCoords_b': ampAdBox.bottom,
-      'frameCoords_l': ampAdBox.left,
+      'frameCoords_t': iframeBox.top,
```

Comment:
Right now the param type for this is !Object.  To ensure proper access, can you update the type to reflect that its a rect?

Reply:
done